pinot-core/src/main/java/org/apache/pinot/core/operator/transform/function/IsFalseTransformFunction.java

```diff
@@ -0,0 +1,42 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+package org.apache.pinot.core.operator.transform.function;
+
+import java.util.List;
+import java.util.Map;
+import org.apache.pinot.common.function.TransformFunctionType;
+import org.apache.pinot.core.operator.ColumnContext;
+
+
+public class IsFalseTransformFunction extends IsTrueTransformFunction {
```

Comment:
I don't think it is a good idea to extend `IsTrueTransformFunction` in order to completely change the semantic.

Instead we should have some kind of `AbstractIsBooleanTransformFunction` that `IsTrueTransformFunction`, `IsFalseTransformFunction`, `IsNotFalseTransformFunction`, etc extend

Reply:
+100 to the abstract class. Four implementations should just implement the Boolean operations.

Reply:
Done.